workflow/Service.go

```diff
@@ -705,6 +705,89 @@ func (service *Service) PostEvent(event *Event) *piazza.JsonResponse {
 	return service.statusCreated(event)
 }
 
+func (service *Service) QueryEvents(jsonString string, params *piazza.HttpQueryParams) *piazza.JsonResponse {
+	format, err := piazza.NewJsonPagination(params)
+	if err != nil {
+		return service.statusBadRequest(err)
+	}
+
+	// if both specified, "by id"" wins
```

Comment:
Not sure if this is necessary... Do we want to let them filter by EventType and supply a DSL query?  Might want to do just the dsl query